CONTRIBUTING.md

```diff
@@ -836,6 +829,20 @@ python setup.py bdist_wheel
 
 First, install dependencies for building docs as described in [Environment Setup and Python configuration](#environment-setup-and-python-configuration).
 
+Building documentation requires [pandoc](https://pandoc.org/index.html), and both
+the pip package and the standalone haskel project are required. Both should have been
+already installed if you used the automated env setup script
+([dev-env-setup.sh](https://github.com/mlflow/mlflow/blob/master/dev/dev-env-setup.sh)),
+but if you manually installing dependencies,
+
+- Pip package: run `pip install pandoc --upgrade`
```

Comment:
Is this really required?

Reply:
I don't have the python pandoc, but can build the docs.

Reply:
Good catch, I just tried same and suceeded building now. Don't quite remember where it came.... tho I think it's good to remove. Supposedly the library is just python binding of it's functionaliry.